src/componentsNext/Filepicker/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { Breadcrumb, BreadcrumbRoot } from "~/design/Breadcrumbs";
+import {
+  Filepicker,
+  FilepickerClose,
+  FilepickerHeading,
+  FilepickerList,
+  FilepickerListItem,
+  FilepickerSeparator,
+} from "~/design/Filepicker";
+import { FolderUp, Home } from "lucide-react";
+import { Fragment, useState } from "react";
+
+import Button from "~/design/Button";
+import { ButtonBar } from "~/design/ButtonBar";
+import Input from "~/design/Input";
+import { analyzePath } from "~/util/router/utils";
+import { fileTypeToIcon } from "~/api/tree/utils";
+import { useNodeContent } from "~/api/tree/query/node";
+import { useTranslation } from "react-i18next";
+
+const FilepickerMenu = ({
+  namespace,
+  defaultPath,
+  onChange,
+}: {
+  namespace: string;
+  defaultPath?: string;
+  onChange: (newValue: string) => void;
+}) => {
+  const [path, setPath] = useState(defaultPath ? defaultPath : "/");
+
+  const givenNamespace = namespace ? namespace : "";
+
+  const { data, isFetched, isError } = useNodeContent({
+    path,
+    givenNamespace,
+  });
+
+  // ... Quick Fix - Error Handling needs to be inside useNodeContent, like for the namespace
+  if (isError) setPath("/");
+  // ...
```

Comment:
If there is an error, you could display a note like "could render a tree, make sure namespace/the-path-to-file exists"